src/avtt/content-script.js

```diff
@@ -0,0 +1,580 @@
+async function postChatMessage({message, color, icon, title, whisper}) {
+    const user = getUser().uid;
+    const room = getRoom();
+    const token = await getAccessToken();
+    const character = getCharacter();
```

Comment:
I made it this way since I have no idea what is the expiry time of an firebase access token and how often it is refreshed. This will at least make sure the token is not expired.